src/JiraCli/ArgumentParser.cs

```diff
@@ -107,6 +107,12 @@ public static Context ParseArguments(List<string> commandLineArguments)
                     continue;
                 }
 
+                if (IsSwitch("issues", name))
+                {
+                    context.Issues = value.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
It appears that the argument parser doesn't support spaces being used within argument values. It splits out argument values based on space:

```
  public static Context ParseArguments(string commandLineArguments)
        {
            return ParseArguments(commandLineArguments.Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries).ToList());
        }
```

So using an argument value like: "test-1, test-2" would throw a spanner in the works, where as "test-1,test-2" is fine.